### PR TITLE
Region sidebar accordion

### PR DESCRIPTION
**Pull request by fsdavid**

### CODE REVIEW

File: src/ui/nehubaContainer/nehubaContainer.template.html [hunk 1]
```diff
@@ -113,6 +113,7 @@
     <!-- sidenav-content -->
     <mat-drawer class="visible col-10 col-sm-10 col-md-5 col-lg-4 col-xl-3 col-xxl-2 d-flex flex-column pe-all"
       mode="push"
+      [ngStyle]="{backgroundColor: selectedTemplate?.useTheme === 'dark'? 'rgba(66,66,66,0.9)' : 'rgba(155,155,155,0.9)'}"
```
Comment (xgui3783) [inline src/ui/nehubaContainer/nehubaContainer.template.html:116]: This is definitely not the right place for styling.

File: src/ui/nehubaContainer/nehubaContainer.template.html [hunk 2]
```diff
@@ -427,6 +344,136 @@
   </div>
 </ng-template>
 
+<ng-template #singleRegionTmpl let-region="region">
```
Comment (xgui3783) [inline src/ui/nehubaContainer/nehubaContainer.template.html:347]: Is there a reason for moving the block of code?

File: src/ui/nehubaContainer/nehubaContainer.component.ts [hunk 3]
```diff
@@ -145,6 +146,8 @@ export class NehubaContainer implements OnInit, OnChanges, OnDestroy {
   @ViewChild('matDrawerMinor', { static: true })
   private matDrawerMinor: MatDrawer
 
+  @ViewChild('regionMenuComponent', {read: RegionMenuComponent}) regionMenuComponent: RegionMenuComponent
```
Comment (xgui3783) [inline src/ui/nehubaContainer/nehubaContainer.component.ts:149]: Not used anywhere?

File: src/ui/nehubaContainer/nehubaContainer.template.html [hunk 4]
```diff
@@ -427,6 +344,136 @@
...
+
+    <!--    Explore in other template -->
+    <mat-expansion-panel [expanded]="expandedSideBarAccordion === 0"
+                         (opened)="expandedSideBarAccordion = 0"
+                         hideToggle
+                         class="mt-1 mb-1"
+                         *ngIf="(regionMenuComponent?.regionInOtherTemplates$ | async)?.length as sameRegionLength">
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          Explore in other templates
```
Comment (xgui3783) [inline src/ui/nehubaContainer/nehubaContainer.template.html:424]: I don't remember exactly, but did we discuss this to be it's own section?

File: src/ui/nehubaContainer/nehubaContainer.template.html [hunk 5]
```diff
@@ -427,6 +344,136 @@
...
+                         hideToggle
+                         *ngIf="selectedParcellation && selectedParcellation.hasAdditionalViewMode
+                         && selectedParcellation.hasAdditionalViewMode.includes('connectivity') && selectedRegions?.length">
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          Connectivity
+        </mat-panel-title>
+        <mat-panel-description class="d-flex align-items-center justify-content-end">
+          <span class="mr-3">{{selectedRegionConnectionNumber}}</span>
+          <i class="fab fa-connectdevelop"></i>
```
Comment (xgui3783) [inline src/ui/nehubaContainer/nehubaContainer.template.html:456]: We need to stop using connect develop icon.

File: src/ui/nehubaContainer/nehubaContainer.template.html (hunk 1 above)
Comment (xgui3783) [reply src/ui/nehubaContainer/nehubaContainer.template.html]: resolved by 0259063

File: src/ui/nehubaContainer/nehubaContainer.template.html (hunk 2 above)
Comment (xgui3783) [reply src/ui/nehubaContainer/nehubaContainer.template.html:346]: resolved by 0259063

File: src/ui/nehubaContainer/nehubaContainer.component.ts (hunk 3 above)
Comment (xgui3783) [reply src/ui/nehubaContainer/nehubaContainer.component.ts]: resolved by 0259063

File: src/ui/nehubaContainer/nehubaContainer.template.html (hunk 5 above)
Comment (xgui3783) [reply src/ui/nehubaContainer/nehubaContainer.template.html]: resolved by 0259063

File: src/ui/nehubaContainer/nehubaContainer.template.html (hunk 4 above)
Comment (fsdavid) [reply src/ui/nehubaContainer/nehubaContainer.template.html]: As I remember yes, I mentioned it at the last meeting and I do not remember the exact answer, but I've understood it as an agreement.

Comment (xgui3783) [reply src/ui/nehubaContainer/nehubaContainer.template.html]: aww, yes this is now resolved

**Pull request merged**